help/de/Help_ch_Account-Actions.xml

```diff
@@ -950,54 +957,72 @@
 
       <para>Die Registerkarte <guilabel>Anfangsbestand</guilabel> ist nur beim Anlegen eines neuen Kontos
         sichtbar, und ist für Konten vom Typ <emphasis>Aktien</emphasis> und
-        <emphasis>Investmentfonds</emphasis> deaktiviert. Im letzteren Fall müssen die
-        Eröffnungssalden von Hand erstellt werden.
+        <emphasis>Investmentfonds</emphasis> deaktiviert.
+<!-- Im letzteren Fall müssen die Eröffnungssalden von Hand erstellt werden. -->
 <!-- FIXME Is there a possibility to refer to a chapter in the "Guide"? -->
+<!--    <ulink url="https://www.gnucash.org/docs/v&series-stable;/de/gnucash-guide/??????">
+        Tutorial und Konzepte, Eröffnungssaldo eines Aktienkontos</ulink>
+        
```

Comment:
The current hack: 
`<ulink url="&url-docs-build-de;-guide/??????">Blah</ulink>`

Reply:
ok.